### PR TITLE
Assorted style changes (PEP8)

### PR DESCRIPTION
**Pull request by kseistrup**
This pull-request contains a slew of style changes suggested by PEP8/pylint:

* Unnecessary parens after ‘if’ keyword removed
* Missing whitespace around arithmetic operator inserted
* Closing bracket doesn't match indentation of opening bracket's line
* Expected 2 blank lines, found 1
* Multiple spaces before operator
* Too many blank lines
* Expected 1 blank line, found 0

The PR doesn't correct all such suggestions.  In particular, parser.py contains a lot of class methods that dont' reference `self` and could be a function instead.  These haven't been touched.

### CODE REVIEW

File: xonsh/ast.py [hunk 1]
```diff
@@ -62,7 +62,7 @@ def __init__(self, parser):
         self.lines = None
         self.mode = None
 
-    def ctxvisit(self, node, input, ctx, mode='exec'):
+    def ctxvisit(self, node, input_string, ctx, mode='exec'):
```
Comment (scopatz) [inline xonsh/ast.py]: would you mind changing this to `inp`, rather than `input_string`?  Not a big deal either way...

File: xonsh/pretty.py [hunk 2]
```diff
@@ -158,14 +159,15 @@ def indent(self, indent):
             self.indentation -= indent
 
     @contextmanager
-    def group(self, indent=0, open='', close=''):
+    def group(self, indent=0, gopen='', close=''):
```
Comment (scopatz) [inline xonsh/pretty.py]: please also change `close` here too.

**Comment by scopatz** [comment]
Thanks a ton for putting this in @kseistrup!  Just a couple of naming issues.

**Comment by kseistrup** [comment]
Would you mind merging it as it is now, and I promise I'll include your suggestions in another pull request that is coming up as soon as this one is accepted.  I can't seem to cherrypick from github's webui.

**Comment by scopatz** [comment]
Sure thing. Merged.

**Comment by kseistrup** [comment]
:)

**Comment by kseistrup** [comment]
Github must have decided to include even the changes I pushed after making the pull-request, so there's only the changes you suggested above in the latest pull-request I submitted a few minutes ago.

**Pull request merged**